recipes/sklearn-crfsuite/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "sklearn-crfsuite" %}
+{% set version = "0.3.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2f59aad3055e01a778a79a6352891cac04788e8b52688aa5bc8b11be7717861e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
```

Comment:
The `setup.py` specifies version restrictions that should be replicated here:

https://github.com/TeamHG-Memex/sklearn-crfsuite/blob/master/setup.py#L15-L19

Reply:
I've added the pins.